metaflow/plugins/kfp/tests/flows/failure_flow.py

```diff
@@ -50,6 +43,9 @@ def download_kubectl(self):
     @retry
     @step
     def user_failure(self):
+        if self.retry_count < 1:
+            raise Exception("start did not retry!")
+
```

Comment:
This looks like a duplicate to code 3 lines later.

Reply:
3 lines later checks on `current.retry_count` rather than `self.retry_count`
I renamed this to `self.start_retry_count` to reduce confusion